converters/fromZigbee.js

```diff
@@ -3598,7 +3598,7 @@ const converters = {
             case tuya.dataPoints.moesSheatingSetpoint:
                 return {current_heating_setpoint: value};
             case tuya.dataPoints.moesSlocalTemp:
-                return {local_temperature: (value / 10)};
+                return {local_temperature: (value / 10).toFixed()};
```

Comment:
I see `toFixed()` is still used here (which will make it a string instead of a number)

Reply:
It worked fine for me as before, but users really wanted to see .0 or .5 after the number, so I made .toFixed (1) that way.

What should I do that number looks like 20.0 or 20.5?